CHANGELOG.md

```diff
@@ -1,5 +1,46 @@
+ * User tag snippets and completion in Qute templates account for the tag parameters. See [redhat-developer/quarkus-ls#784](https://github.com/redhat-developer/quarkus-ls/issues/784).
+ * Validatation and go to definition for user tag parameters in Qute templates. See [redhat-developer/quarkus-ls#788](https://github.com/redhat-developer/quarkus-ls/issues/788), [redhat-developer/quarkus-ls#789](https://github.com/redhat-developer/quarkus-ls/issues/789).
+ * CodeAction to insert required user tag parameters in Qute template. See [redhat-developer/quarkus-ls#797](https://github.com/redhat-developer/quarkus-ls/issues/797).
+ * Make Java file snippets context-aware. See [redhat-developer/quarkus-ls#782](https://github.com/redhat-developer/quarkus-ls/issues/782).
+ * Add support for Qute `#fragment` section. See [redhat-developer/quarkus-ls#768](https://github.com/redhat-developer/quarkus-ls/pull/768), [redhat-developer/quarkus-ls#769](https://github.com/redhat-developer/quarkus-ls/pull/769).
+ * When applying the Qute template quick fix to generate a missing class member, generate the member at the end of the class. See [redhat-developer/quarkus-ls#692](https://github.com/redhat-developer/quarkus-ls/pull/692).
+ * Support for validation, completion and document link for `#include` and `#insert` sections in Qute templates. See [#438](https://github.com/redhat-developeredhat-developer/quarkus-lsr/quarkus-ls/issues/438).
+ * "Show references" CodeLens for `#insert` section in Qute templates. See [#595](https://github.com/redhat-developer/vscode-quarkus/pull/595).
+ * Support surround with section/comments command for Qute templates. See [#581](https://github.com/redhat-developer/vscode-quarkus/issues/581).
+ * Provide syntax coloration for #fragment. See [#570](https://github.com/redhat-developer/vscode-quarkus/pull/570).
+
+### Bug Fixes
+
+ * Treat `CompletionStage` and `Uni` objects as their resolved type for validation and completion in Qute templates. See [redhat-developer/quarkus-ls#826](https://github.com/redhat-developer/quarkus-ls/issues/826).
+ * `{#if}` sections in Qute templates of the form `{#if !inject:beanName}{/if}` are marked as invalid. See [redhat-developer/quarkus-ls#828](https://github.com/redhat-developer/quarkus-ls/issues/828).
+ * Quick fix to generate a `@TemplateExtension` class from a Qute template might place the class in the wrong folder. See [redhat-developer/quarkus-ls#831](https://github.com/redhat-developer/quarkus-ls/issues/831).
+ * "Create project" fails when there are no folders and one java file open. See [#559](https://github.com/redhat-developer/vscode-quarkus/issues/559).
+ * Fix Quarkus debug configuration after maven wrapper is deleted. See [#572](https://github.com/redhat-developer/vscode-quarkus/issues/572).
+ * qute-ls isn't stopped by vscode-quarkus. See [#576](https://github.com/redhat-developer/vscode-quarkus/issues/576).
```

Comment:
Maybe rename to something like:
`qute-ls remains running after vscode-quarkus is stopped`